service-base/src/main/java/org/eclipse/hono/service/auth/ExternalJwtAuthTokenValidator.java

```diff
@@ -52,6 +52,16 @@ public class ExternalJwtAuthTokenValidator implements AuthTokenValidator {
     private static final String EXPECTED_TOKEN_TYPE = "JWT";
     private CredentialsObject credentialsObject;
 
+    /**
+     * Adds the allowed clock skew to the provided expiration time.
+     *
+     * @param exp The expiration time without the allowed clock skew.
+     * @return The expiration time with the allowed clock skew.
```

Comment:
thanks for adding JavaDoc :+1: 
Can you also please add a throws clause for the NPE that will be thrown if the passed-in instant is `null`?

Reply:
sure, done.